tests/core/test_inventory.py

```diff
@@ -236,3 +236,43 @@ def test_children_of_obj(self):
             inv.hosts["dev4.group_2"],
             inv.hosts["dev3.group_2"],
         }
+
+    def test_add_host(self):
```

Comment:
Nice! Would you mind adding also some connection_options  to h3 as that's the tricky part here?

Reply:
Thank you. I added the assertions to both add_host and add_group tests.